siliconcompiler/client.py

```diff
@@ -329,7 +337,10 @@ def fetch_results(chip):
     job_hash = chip.status['jobhash']
     local_dir = chip.get('option', 'builddir')
 
-    # Authenticated jobs get a zip file full of other zip files.
+    if results_failed:
+        chip.error("Sorry, something went wrong and your job results could not be retrieved.", fatal=True)
```

Comment:
I think it's fine to leave out the Sorry.
On failure should we avoid deleting the job?
Is there an alternative method they could retrieve the results?